mk/spksrc.tc-vers.mk

```diff
@@ -1,61 +1,65 @@
 ifeq ($(strip $(TC_OS_MIN_VER)),)
-TC_OS_MIN_VER = $(TC_FIRMWARE)
+TC_OS_MIN_VER = $(TC_OS_MIN_VER)
 endif
 
-ifeq ($(strip $(TC_FIRMWARE)),)
+ifeq ($(strip $(TC_OS_MIN_VER)),)
 ifeq ($(TC_VERS),1.1)
-TC_FIRMWARE = $(TC_VERS)-6931
+TC_OS_MIN_VER = $(TC_VERS)-6931
 endif
 
 ifeq ($(TC_VERS),1.2)
-TC_FIRMWARE = $(TC_VERS)-7742
+TC_OS_MIN_VER = $(TC_VERS)-7742
 endif
 
 ifeq ($(TC_VERS),4.0)
-TC_FIRMWARE = $(TC_VERS)-2198
+TC_OS_MIN_VER = $(TC_VERS)-2198
 endif
 
 ifeq ($(TC_VERS),4.1)
-TC_FIRMWARE = $(TC_VERS)-2636
+TC_OS_MIN_VER = $(TC_VERS)-2636
 endif
 
 ifeq ($(TC_VERS),4.2)
-TC_FIRMWARE = $(TC_VERS)-3202
+TC_OS_MIN_VER = $(TC_VERS)-3202
 endif
 
 ifeq ($(TC_VERS),4.3)
-TC_FIRMWARE = $(TC_VERS)-3776
+TC_OS_MIN_VER = $(TC_VERS)-3776
 endif
 
 ifeq ($(TC_VERS),5.0)
-TC_FIRMWARE = $(TC_VERS)-4458
+TC_OS_MIN_VER = $(TC_VERS)-4458
 endif
 
 ifeq ($(TC_VERS),5.1)
-TC_FIRMWARE = $(TC_VERS)-5004
+TC_OS_MIN_VER = $(TC_VERS)-5004
 endif
 
 ifeq ($(TC_VERS),5.2)
-TC_FIRMWARE = $(TC_VERS)-5644
+TC_OS_MIN_VER = $(TC_VERS)-5644
 endif
 
 ifeq ($(TC_VERS),6.0)
-TC_FIRMWARE = $(TC_VERS)-7321
+TC_OS_MIN_VER = $(TC_VERS)-7321
 endif
 
 ifeq ($(TC_VERS),6.0.2)
-TC_FIRMWARE = $(TC_VERS)-8451
+TC_OS_MIN_VER = $(TC_VERS)-8451
 endif
 
 ifeq ($(TC_VERS),6.1)
-TC_FIRMWARE = $(TC_VERS)-15047
+TC_OS_MIN_VER = $(TC_VERS)-15047
 endif
 
 ifeq ($(TC_VERS),6.2)
-TC_FIRMWARE = $(TC_VERS)-22259
+TC_OS_MIN_VER = $(TC_VERS)-22259
 endif
 
 ifeq ($(TC_VERS),6.2.2)
-TC_FIRMWARE = $(TC_VERS)-24922
+TC_OS_MIN_VER = $(TC_VERS)-24922
```

Comment:
should be TC_OS_MIN_VER = 6.2-24922

I can't remember, but it can be that you sad to fix this in a future branch.
(similar below for 6.2.3)

Reply:
humm, interesting.  Indeed perhaps I could strip from `x.y.z` any subsequent `.z` in order to always `x.y-build#`.
But would that make sense?  I mean we would then have:
```
6.2-22259
6.2-24922
6.2-25423
```
Instead of:
```
6.2-22259
6.2.2-24922
6.2.3-25423
```

Reply:
Yes absolutly. I posted that the os_min_ver setting is defined as this. Synology uses this scheme and the repos must too.

So when will build for TC 6.2.2 or TC 6.2.3 the packages will be named with the full DSM Version. But the internally used version in the INFO file is defined by os_min_ver of the format X.Y-Z (see synology development guide - I verified only with the DSM 7 preview Guide, but this was always the same).

Reply:
Ok then I'll create a new PR to address this specifically.  I'll probably have to add a new `TC_BUILD_VER` variable (or similar) within `spksrc.tc-vers.mk` for each version and then generate the final `TC_OS_MIN_VER` in `spksrc.tc-flags.mk`...  I'll try to address this over the week (actually before we start submitting new packages online).

Reply:
Hopefully addressed with PR #4219